IntegrationTests/ReducedCombinedConfig/script/MemPrints

```diff
@@ -0,0 +1 @@
+../../../emData/MemPrintsReducedCM
```

Comment:
@aryd Please remove this symlink, since it is generated by compileHLS.tcl.

Reply:
Removed